kafka-client/src/main/java/dev/responsive/kafka/internal/db/rs3/RS3TableFactory.java

```diff
@@ -92,6 +92,38 @@ public RemoteKVTable<WalEntry> kvTable(
     );
   }
 
+  public RemoteWindowTable<WalEntry> windowTable(
+      final String name,
+      final ResponsiveConfig config,
+      final ResponsiveMetrics responsiveMetrics,
+      final ResponsiveMetrics.MetricScopeBuilder scopeBuilder
+  ) {
+    final var storeId = lookupStoreId(config, name);
+    final var pssPartitioner = new PssDirectPartitioner();
+    final var rs3Client = connector.connect();
+    return new RS3WindowTable(
+        name,
+        storeId,
+        rs3Client,
+        pssPartitioner,
+        responsiveMetrics,
+        scopeBuilder
+    );
+  }
+
+  private UUID lookupStoreId(
```

Comment:
fyi i'm just about to merge the `rs3` PR to convert store ids from UUID to string, haven't done the `responsive-pub` one yet but it probably makes sense to just drop the UUID and use string for the window store now so you don't have to rebase this